buf.gen.yaml

```diff
@@ -8,6 +8,6 @@ managed:
     - file_option: go_package_prefix
       value: github.com/bufbuild/protoschema-plugins/internal/gen/proto
 plugins:
-  - remote: buf.build/protocolbuffers/go
+  - remote: buf.build/protocolbuffers/go:v1.36.1
```

Comment:
Updated to pin this for reproducible builds. The `upgrade` make target will bump this if the protobuf-go dependency is updated.